.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Fixes #<!-- If this addresses a specific issue, please provide the issue number here -->
+
+- [ ] CHANGELOG.md updated
+
+## Description
+<!-- Please discuss the changes you have worked on. What do the changes do; why is this PR needed? -->
+
+## Specific Changes
+<!-- Please list the changes in a concise manner. -->
+
+  -
+  -
+  -
+
+## Testing
+<!-- If you are adding a new feature to a library, you must include tests for your new code. -->
```

Comment:
This one looks like a checkbox than a section.

Or we can also turn this one into screenshots of your great work.

Your call.